NoSoft-SA/nspack
# active tripsheet pallets

Summary of this change:
add check to reworks (single and bulk edit, also scrap) to disallow action if pallet is on an active tripsheet.
already on reject-to-repack

### Added
- . check to reworks (single and bulk edit, also scrap) to disallow action if pallet is on an active tripsheet.

How to test this code via the UI:
Production | Reworks | Single Pallet Edit | New
* should block if pallet is on an open tripsheet
